pkg/nuclide/flow/lib/FlowLinter.js

```diff
@@ -71,12 +74,10 @@ function processDiagnostics(diagnostics: Array<Object>, targetFile: string) {
   };
 
   // Filter messages not addressing `targetFile` and merge messages spanning multiple files.
-  var messages = diagnostics.map( (diagnostic) => {
-                  var diagnosticMessages = diagnostic['message'];
-                  return mergeFlowMessages(diagnosticMessages);
-                 }).filter(hasMessageWithPath);
-
-  return messages.map(flowMessageToLinterMessage);
+  return diagnostics
+    .map( (diagnostic) => diagnostic['message'] )
+    .filter(hasMessageWithPath)
```

Comment:
Each message is an array of objects. `hasMessageWithPath` expects an object with a `path` property. So, it always returns false and all linter messages are filtered out.